guides/doc-Content_Management_Guide/topics/Managing_Errata.adoc

```diff
@@ -310,20 +307,18 @@ Use these procedures to review and apply errata to a host.
 * Synchronize {ProjectName} repositories with the latest errata available from Red{nbsp}Hat.
 For more information, see xref:Importing_Content-Synchronizing_Repositories[].
 
-* Register the host to an environment and Content View on {ProjectServer}.
+* Register the host to an environment and Content View on the {ProjectServer}.
 For more information, see {ManagingHostsDocURL}Registering_Hosts[Registering Hosts] in the _Managing Hosts_ guide.
 
-* For RHEL 7 hosts, ensure that you install the `katello-agent` package.
-For more information, see {ManagingHostsDocURL}installing-the-katello-agent_managing-hosts[Installing the Katello Agent] in the _Managing Hosts_ guide.
-+
-Note that the Katello agent is deprecated and will be removed in a future {Project} version.
-Migrate your workloads to use the remote execution feature to update clients remotely.
-For more information, see {ManagingHostsDocURL}host-management-without-goferd-and-katello-agent_managing-hosts[Host Management Without Goferd and Katello Agent] in the _Managing Hosts Guide_.
-
-.For Red{nbsp}Hat Enterprise Linux 8
-To apply an erratum to a RHEL 8 host, you can run a remote execution job on {ProjectServer} or update the host.
+* Configure the host for remote execution.
 For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs] in the  _Managing Hosts_ guide.
 
+[NOTE]
+====
+If the host is already configured to receive content updates with the deprecated Katello Agent, they should be migrated to remote execution instead.
```

Comment:
```suggestion
If the host is already configured to receive content updates with the deprecated Katello Agent, migrate to remote execution instead.
```

I'm not sure who "them" is, whether it is the host or the content updates. 
Docs style guide doesn't like the word should and prefers the imperative :)